src/figuresClasses.ts

```diff
@@ -1,11 +1,89 @@
-export interface Figure {}
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+type Color = 'red' | 'green' | 'blue';
+
```

Comment:
Why do you use enum and union type together?

Both work, but please be consistent and pick one.